src/main.rs

```diff
@@ -1,43 +1,50 @@
-use std::{error::Error, fs::File, io::{Read, self, BufReader}, mem::size_of, u8, usize, os::unix::prelude::{AsFd, FromRawFd, AsRawFd}};
+use std::{
```

Comment:
Let's find a rust lint action and add it to the GitHub workflow for any pull requests onton main so that branches get checked for lint issues.

Reply:
see https://github.com/sbates130272/iowatcher-ng/pull/7